spec/python-spec.coffee

```diff
@@ -278,3 +278,28 @@ describe "Python grammar", ->
     expect(tokens[4][0]).toEqual value: ')', scopes: ['source.python', 'meta.function.python', 'punctuation.definition.parameters.end.python']
     expect(tokens[4][1]).toEqual value: ':', scopes: ['source.python', 'meta.function.python', 'punctuation.section.function.begin.python']
 
+
+  it "correctly enables SQL inline highlighting on blocks", ->
+    tokens = grammar.tokenizeLines('''
```

Comment:
Wouldn't that complicate the spec somewhat? I've written it up and it works, I find it less readable than before though.